### ISSUE TITLE
Build Light Curves

### ISSUE THREAD

**Issue by jradavenport** [jradavenport/FFIorBUST#5]
After the photometry is run, we need to create light curves. To do this, _sources_ (per exposure) must be matched between images, thus determining the unique _objects_ observed.

This list is less deterministic, and will depend on the approach/code we use to match between exposures. Please edit accordingly and add notes!
- [ ] match data between frames
- [ ] match data to the Kepler Input Catalog (kic.txt.gz) to get $grizJHK$ colors and object ID's

**Comment by hover2pi** [comment]
The code SHOULD already do both. I'm going to make a super-scaled-down version of the data to test.

**Comment by bsipocz** [comment]
We can talk about this during the telecon, but I think we should primarily do the first one, and use KIC as extra info (it doesn't have the transients in it, etc.)

**Comment by jradavenport** [comment]
I agree with this, that we should match the frames to each other first, then match resulting objects to the KIC data
